lib/bigcommerce/resources/v3/email_template.rb

```diff
@@ -0,0 +1,13 @@
+module Bigcommerce
+  module V3
+    class EmailTemplate < Resource
+      include Bigcommerce::ResourceActions.new uri: 'marketing/email-templates/%s'
+
+      property :type_id
+      property :subject
+      property :body
+      property :translations
```

Comment:
@william1benn I think you can add `coerce: Array` param to this property, so it will be automatically handled as an array